src/utils/components/CPUMemoryModal/hooks/useTemplateDefaultCpuMemory.ts

```diff
@@ -38,9 +39,7 @@ const useTemplateDefaultCpuMemory: UseTemplateDefaultCpuMemory = (
       });
   }, [templateName, templateNamespace]);
 
-  const defaultMemory = getMemorySize(
-    template?.objects?.[0]?.spec?.template?.spec?.domain?.memory?.guest,
-  );
+  const defaultMemory = getMemorySize(getMemory(template?.objects?.[0]));
```

Comment:
Please use getTemplateVirtualMachineObject method instead of `template?.objects?.[0]` here and below

Reply:
now saw that you also have getTemplateVirtualMachineCPU